### ISSUE TITLE
[Feature Request]: Bulk Add / Custom Field per Asset Tag

### ISSUE THREAD

**Issue by Venoxi24** [grokability/snipe-it#15391]
### Is your feature request related to a problem? Please describe.

Mein Problem ist, dass ich eine Geräte Kategorie "Smartphone / Ipad " anglegt habe, welche das Custom Field IMEI haben. Wenn ich nun einen neuen Asset hinzufüge und die Kategorie auswähle, dann erscheint weiter unten mein Feld mit der IMEI. Wenn ich aber oben nun mit dem "+" weitere Asset Tags zufüge, kommt unter der zweiten, dritten... Asset Tag auch das feld mit der Seriennummer, aber es wird egal wo kein Feld für eine IMEI hinzugefügt. 

### Describe the solution you'd like

I would like to have it so that if a custom field is unique per device, this custom field is then displayed at the top under the asset tag and the serial number, just like a serial number.

**Comment by snipe** [comment]
Please post in English moving forward.

> My problem is that I have created a device category “Smartphone / Ipad ”, which has the custom field IMEI. If I now add a new asset and select the category, my field with the IMEI appears further down. However, if I now add further asset tags with the “+” at the top, the field with the IMEI also appears under the second, third... Asset Tag, the field with the serial number also appears, but no field for an IMEI is added anywhere.
> 
> Translated with DeepL.com (free version)

**Comment by snipe** [comment]
I'm not clear on what you're describing here. Can you provide screenshots or video?

**Comment by Venoxi24** [comment]
Okay here is a slightly different description:

![Screenshot 2024-08-28 151934](https://github.com/user-attachments/assets/8e21a4a8-9fe0-4f33-a05a-b63a0c3621d1)

If the option is activated for a custom field, I would like it if the custom field is also defined per asset and not for all assets when creating assets at the same time, since this must be unique

![Screenshot 2024-08-28 152226](https://github.com/user-attachments/assets/9afd7aa9-dd99-4bda-a84b-7f22c3098af7)

**Comment by neokleon** [comment]
Yup this is a a trick request as assets cant be managed in quantity.
You can do a workaround
create you desired asset completely
then export it via export feature with all parameters
fill out the excel csv sheet with other asset information
import it in snipe-it

this work around is the best option i think....

**Comment by Venoxi24** [comment]
Hi, yes I think this would work, but it would be nice if fields that have to be unique per asset could be entered per asset tag. If you create multiple assets and then fill in a field in the current version, the system returns a 500 error.
